ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -31,4 +31,25 @@ extension View {
 
         return AnyView(self)
     }
+
+    /// Measures the size of a view, monitors when its size is updated, and takes a closure to be called when it does
+    /// - Parameter onChange: Block to be performed on size change
+    /// - Returns The modified view.
+    func measureSize(onChange: @escaping (CGSize) -> Void) -> some View {
+      background(
```

Comment:
Can you create a struct that implements ViewModifier and just apply the it in the logic of this func? (and name it OnSizeChangeViewModifier as per the comment above)

Reference:
https://github.com/microsoft/fluentui-apple/blob/57bbca9b3a48078b1bab6b46907548aa439110bd/ios/FluentUI/Vnext/HUD/HUDModifiers.swift#L52

Reply:
Sure! Though I'm curious as to why, is it just easier to read or is there a performance difference?

Reply:
Not for any perf difference as far as I'm aware. Although I've seen clipping animations not working property in the past when using a func returning the clip shape as opposed to just creating the shape in a View struct.

I believe SwiftUI needs to keep track of the structs to decide what to refresh/redraw or animate.

Not too long ago we decided to follow the standard which is creating these modifiers as structs that implement the protocol and leaving the func just as an easy way to implement it.